examples/channels-correlation/channels_correlation.description

```diff
@@ -0,0 +1,4 @@
+// Channel is a constrained type that can be defined only as a top level node.
+// It is introduced for communication between parallel processes in Ballerina programs.
+// Channels can be used for message correlation by sending and receiving messages via different resources to the same channel.
```

Comment:
Shall we change the sentence as follows:
"// If you want to send and receive messages via different resources to the same channel, you can use channels to correlate messages."

Reply:
Well, the expected meaning here is that, if you need to do message correlation, you can do that by sending and receiving messages between different resources to the same channel. Your suggestion implies other way around right?

Reply:
Yeah. The suggestion implies the other way around. Let's keep it as it is